Description: Disable the loom feature+dependency
 which depends on generator, which fails to build on everything except amd64
 and arm64
Last-Update: 2024-09-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -64,4 +64,4 @@
 
-[target."cfg(loom)".dependencies.loom]
-version = "0.5"
-optional = true
+#[target."cfg(loom)".dependencies.loom]
+#version = "0.5"
+#optional = true
